gammapy/time/lightcurve.py

```diff
@@ -68,3 +69,35 @@ def simulate_example(cls):
         lc['FLUX_ERR'] = Quantity([0.1, 0.4, 0.7, 0.9], 'cm^-2 s^-1')
 
         return lc
+
+    def compute_fvar(self):
+        """Calculate the fractional excess variance (Fvar) of `LightCurve`.
+
```

Comment:
Please add a link to a paper or webpage with a description / formula for this.

Maybe also mention which columns from the lightcurve are accessed.

Also put this in the docstring:
```
Returns
--------
fvar, fvarerr : float
    Normalised excess variance and error on it.
```
```